mycore-mods/src/main/resources/xslt/solr/indexing/mods-solr.xsl

```diff
@@ -219,9 +219,11 @@
         <field name="mods.relatedItem">
           <xsl:value-of select="$relatedID" />
         </field>
-        <field name="mods.relatedItem.{$type}">
-          <xsl:value-of select="$relatedID" />
-        </field>
+        <xsl:if test="fn:string-length($type) &gt; 0">
```

Comment:
Out of interest: Is the `fn:`-prefix necessary here? Afaik, standard functions don't need a prefix.

Reply:
This was automatically added by my IDE.